core/templates/dev/head/pages/base.html

```diff
@@ -129,8 +136,19 @@
         <div class="oppia-content-container">
           <div id="wrapper">
             <div class="oppia-main-body">
+              <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
+                <div class="navbar-container">
+                  <top-navigation-bar></top-navigation-bar>
 
-              <top-navigation-bar></top-navigation-bar>
+                  <div class="collapse navbar-collapse ng-cloak">
+                    {% block navbar_breadcrumb %}
```

Comment:
Because it actually didn't work! The blocks need to be in the main template file in order for them to be replaceable by other templates that extend base.html.